plugin/EstateList.php

```diff
@@ -735,8 +734,10 @@ public function formatPrice(string $str, string $language, string $locate): stri
 	public function getParkingName(string $parkingName, int $count): string
 	{
 		$str = preg_replace('/([a-z])([A-Z])/', "\\1 \\2", $parkingName);
-		if ($count > 1) {
-			$str .= 's';
+		if (1 === $count) {
```

Comment:
a switch/case statement would be better here, so that it can match on each possible `$parkingName`, so that the strings that need to be translated can be string _literals_ in the code. That is important, otherwise the script that generates the _.pot_ file can't find the strings.

Reply:
@jayay . I updated it